src/plugins/intel_gpu/src/graph/select.cpp

```diff
@@ -25,6 +25,9 @@ layout select_inst::calc_output_layout(select_node const& node, kernel_impl_para
         auto input1_size = impl_param.get_input_layout(1).get_tensor();
         auto input2_size = impl_param.get_input_layout(2).get_tensor();
         output_size = tensor::max(input1_size, input2_size);
+        // Cond input0 also can be broadcasted.
+        auto input0_size = impl_param.get_input_layout(0).get_tensor();
```

Comment:
Why input0, which is selecting from input1 and input 2, can be larger than input1/input2?

Reply:
OpenVINO op runtime reference also allows broadcast on input0. https://github.com/openvinotoolkit/openvino/blob/52b02c190b4a996b6fb711bbb308b39b736ea912/src/core/reference/include/ngraph/runtime/reference/autobroadcast_binop.hpp#L325